### ISSUE TITLE
[TODO List] Maintenance needed (updates to old references, inconsistencies, missing text, multi-genere audit, small rewording)

### ISSUE THREAD

**Issue by jcte02** [openlegend/core-rules#242]
**LAST EDIT ON 09/16/2017**

Since I'm running short on time, I post here my issue list as reference in the hope someone can fix them.

**NOTE** Some of the item listed may not be issues, in that case I will remove them from the list.

- [ ] Now that banes cannot stack, the line `If the same target is affected by this bane from multiple sources, as long as their attack includes one of those who targeted them, they are not affected by the penalty.` in `Provoked` bane is useless. Either this line is removed or it should be specified that this bane can indeed stack.
- [x] ~~`Surprised` bane line `Surprised characters cannot take any actions that are interrupt actions.` should be changed to `Surprised characters cannot take any interrupt actions.` to simplify the language (imho).~~
- [x] ~~`Resistance` boon still references `finesse` and `forceful` damage, but I think this is now obsolete. It could be removed or changed to `precise` and `forceful`.~~
- [x] ~~`Restoration` special note `your roll total must be double the banes normal Power Level. So, a Power Level 9 bane can be dispelled on a roll of 46. ` makes no sense (at least to me). It is trying to say that the roll must be higher that the CR calculated on power level using the boon rules? However, the only way to get 46 as a result is `10 + 4 x the banes Power Level`.~~
- [ ] In `Shapeshift` special note, `Perception attribute` should be changed to `Perception action roll`, because I don't think an action roll should be opposed by a raw attribute score.
- [x] ~~I think that for clarity we should change in `Resistance` from `Your defense scores` to `All of your defenses scores`~~.
- [ ] Power Level 7 of `Telepathy` specifies `creatures that you can see`. This should be propagated to the lower power levels, since I think that is what is intended.
- [x] ~~`Phantasm` bane needs rewording since at the moment it can only be used on a square grid, it should be reworded to mention something like `there should be non disconnected areas` (i.e. no holes, there cannot be two separated immages) which is in line with what it says currently. (see #229).~~
- [x] ~~In `Vicious Strike`, `attack action roll` should be changed to `damaging action roll` (I think).~~
- [x] ~~For the sake of consistency, all references to `Resist roll` or `resist roll` should be updated to `Resist action`.~~
- [x] ~~References to `roll 2 d20 and take the highest result` should be updated to someting like `roll a d20 with advantage` or `you have advantage on the d20 roll`. List of all said references:~~
  - ~~`Vicious Strike`~~
  - ~~`Resilient`~~
  - ~~`Hospitaler`~~
  - ~~`Potent Bane` (`roll the d20 with disadvantage`)~~
- [x] ~~Line `Whenever you take this feat, you must specify which type of attack you are focusing on, and you may take the feat multiple times to specialize in multiple types of attack.` in `Multi-Target Attack Specialist (I - V)` is probably a duplicate of the preceding line and should be removed or modified.~~
- [x] ~~In `Persistent Damage`, `resist roll with advantage 1` is a bit cryptic I think.~~

Now for some multi-genere audit:
- [x] ~~In `Flight`,  `magical force` should be changed to something like `an extraordinary force`.~~
- [x] ~~In `Insubstantial`, `non-magical speed` can be changed to `non-extraordinary speed`.~~
- [x] ~~In `Invisible`, `magical control of light` should be changed to `extraordinary control of light`.~~
- [x] ~~In `Light`, `magical light` should be changed to `extraordinary light`.~~
- [ ] In `Teleport`, `magical or normal means` should be changed to `extraordinary or normal means`.
- [x] ~~There are other issues that I forgot to track, but they can be found by simply searching `magic` in the documents (feats, banes, boons). Mind that not every match is an issue.~~

~~The next is a small rewording that I think is in order, but I'm not a native English speaker so mind that this can be as well not an isssue:~~
- [x] ~~In `Reading`, `you ask` should be changed to `you ascertain`.~~

**TL;DR** Yes it's a long list.
**TO ORGANIZATOS** I don't think this is OT, but if it is then simply close this issue.
**TO CONTRIBUTORS** Please reference this issue in the PRs or the commits that close the issues so it is easier for me to update the list (or simply update it yourself).

As suggested by @Odog4ever, Chapter 4 needs multi-genre audit for:
- [x] ~~Sample lifestyle / goods~~
- [x] ~~Adventuring Gear/Packs~~

**Comment by brianfeister** [comment]
Thanks so much for all your help @jcte02 !

I know it might seem like you're the only one watching these things, that's just because we have time before we go to print and "lock" everything.

Don't burn yourself out and do keep in mind that we have months to correct these issues and, in case it wasn't clear, I will be producing an Alpha Pre-Release of the core rulebook in a few month and sending that out to our ~1,500 backers and soliciting proofreading for things like this from the larger audience.

In the meantime, these sound mostly correct and I'll get to these when time allows.

**Comment by jcte02** [comment]
Lol I know I sounded a bit tragic (for me it was like 4 am when I wrote) but I know that I am not the only one watching. Since I almost read the entire book line per line to update the SRD, I decided to maintain a list and post it as reference also for me later on. Currently I'm on holidays so it's no big deal for me time-wise doing this.

**Comment by brianfeister** [comment]
@jcte02 - I've streamlined the language and rules wording so we can now say "advantage 1" and apply it to any type of roll:
https://github.com/openlegend/core-rules/commit/f736dcbfe9ff80176b5557bb7d90badd9c969842

**Comment by Odog4ever** [comment]
@jcte02 

multi-genre audit for Wealth-Equipment Chapter:

- [ ] Sample lifestyles/goods
- [ ] Adventuring Gear/Packs
